diff --git a/pytest_sugar.py b/pytest_sugar.py
index ca336da..53ecc0c 100644
--- a/pytest_sugar.py
+++ b/pytest_sugar.py
@@ -165,9 +165,9 @@ def pytest_configure(config):
         except ImportError:
             pass
         else:
-            from distutils.version import LooseVersion
-            xdist_version = LooseVersion(xdist.__version__)
-            if xdist_version >= LooseVersion('1.14'):
+            from packaging import version
+            xdist_version = version.Version(xdist.__version__)
+            if xdist_version >= version.Version('1.14'):
                 config.pluginmanager.register(DeferredXdistPlugin())
 
     if IS_SUGAR_ENABLED and not getattr(config, 'slaveinput', None):
